EnglishFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/77313
+bhaskar.com,divyabhaskar.co.in#?#div[id^="Ad--"]
```

Comment:
Better under this line: https://github.com/AdguardTeam/AdguardFilters/blob/1d5893c3d4d19b52f0247b42b1f03a7d07471530/EnglishFilter/sections/foreign.txt#L3549

Reply:
added few additional rules(mobile apps, social, ad without ext css)

Reply:
Fixed.